### PR TITLE
Update release notes with tuple member annotation

### PR DESCRIPTION
**Pull request by ushirask**
## Checklist

- [ ] **Page addition**
  - [ ] Add `permalink` to pages.

- [ ] **Page removal**
  - [ ] Remove entry from corresponding left nav YAML file.
  - [ ] Add `redirect_from` on the alternative page.
  - [ ] If no alternative page, add redirection on the `redirections.js ` file.

- [ ] **Page rename**
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

- [ ] **Page restrcuture**
  - [ ] Add `permalink` to pages.
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

### CODE REVIEW

File: downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md [hunk 1]
```diff
@@ -43,6 +43,18 @@ A few backward-incompatible changes have been introduced during the Swan Lake Be
 
 ### New features
 
+#### Tuple member annotations 
+
+- Added support for annotating tuple members.
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md:48]:
```suggestion
- Added support for annotating tuple members
```

File: downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md [hunk 2]
```diff
@@ -43,6 +43,18 @@ A few backward-incompatible changes have been introduced during the Swan Lake Be
...
+- Tuple members can be annotated with annotations with `field` attachment points
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md:49]:
```suggestion
- Tuple members can be annotated using annotations with `field` attachment points
```

File: downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md [hunk 3]
```diff
@@ -43,6 +43,18 @@ A few backward-incompatible changes have been introduced during the Swan Lake Be
...
+- Similar to records, annotations are not allowed on the rest type of a tuple.
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md:50]:
```suggestion
- Similar to records, annotations are not allowed on the `rest` type of a tuple
```

**Review by praneesha — APPROVED** [review]

File: downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md [hunk 4]
```diff
@@ -43,6 +43,18 @@ A few backward-incompatible changes have been introduced during the Swan Lake Be
 
 ### New features
 
+#### Tuple member annotations 
+
+- Added support for annotating tuple members
+- Tuple members can be annotated using annotations with `field` attachment points
+- Similar to records, annotations are not allowed on the `rest` type of a tuple
```
Comment (MaryamZi) [inline downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md:50]: @ushirask, can you please update this to be in like a paragraph form. We don't usually use point form when describing a feature.

Comment (ushirask) [reply downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md:50]: Updated in #6320

**Pull request merged**